scripts/pre_commit_linter.py

```diff
@@ -243,7 +300,10 @@ def _pre_commit_linter():
         if os.path.isfile(input_path):
             all_files = [input_path]
         else:
-            all_files = _get_all_files_in_directory(input_path)
+            excluded_glob_patterns = _get_glob_patterns_excluded_from_jscsrc(
+                jscsrc_path)
+            all_files = _get_all_files_in_directory(input_path,
```

Comment:
FYI (optional): this works, but an alternative is to break after the '(' and indent by 4. Otherwise the code tends to get a bit squashed up against the right side of the screen.